Source/WebCore/Modules/WebGPU/GPUAdapter.idl

```diff
@@ -37,4 +37,5 @@ interface GPUAdapter {
     readonly attribute boolean isFallbackAdapter;
 
     [CallWith=CurrentScriptExecutionContext] Promise<GPUDevice> requestDevice(optional GPUDeviceDescriptor descriptor);
+    [CallWith=CurrentScriptExecutionContext] Promise<undefined> requestAdapterInfo(optional sequence<DOMString> unmaskHints = []);
```

Comment:
Do we really need the script execution context for this?

Reply:
Not at this point, I will remove it